Makefile

```diff
@@ -144,8 +144,11 @@ test-integration: manifests generate fmt vet envtest ginkgo ## Run tests.
 	$(GINKGO) --junit-report=junit.xml --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET)
 
 USE_EXISTING_CLUSTER ?= false
-.PHONY: test-e2e-kind
-test-e2e-kind: kustomize manifests generate fmt vet envtest ginkgo kind-image-build
```

Comment:
If you do change the name of this (which is fine!), then please update this  also:

https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/kueue/kueue-presubmits.yaml#L53

Reply:
sorry missed that! Will open a PR there as well